pkgs/tools/admin/google-cloud-sdk/gsutil-disable-updates.patch

```diff
@@ -0,0 +1,147 @@
+diff -ru google-cloud-sdk.orig/platform/gsutil/gslib/command_runner.py google-cloud-sdk/platform/gsutil/gslib/command_runner.py
+--- google-cloud-sdk.orig/platform/gsutil/gslib/command_runner.py	2019-10-17 15:41:04.000000000 +0000
++++ google-cloud-sdk/platform/gsutil/gslib/command_runner.py	2020-02-13 10:51:28.474801743 +0000
+@@ -61,7 +61,6 @@
+ from gslib.utils.text_util import print_to_fd
+ from gslib.utils.unit_util import SECONDS_PER_DAY
+ from gslib.utils.update_util import LookUpGsutilVersion
+-from gslib.tests.util import HAS_NON_DEFAULT_GS_HOST
+ 
+ 
+ def HandleHeaderCoding(headers):
+@@ -331,17 +330,6 @@
+     Returns:
+       Return value(s) from Command that was run.
+     """
+-    command_changed_to_update = False
+-    if (not skip_update_check and
+-        self.MaybeCheckForAndOfferSoftwareUpdate(command_name, debug)):
+-      command_name = 'update'
+-      command_changed_to_update = True
+-      args = [_StringToSysArgType('-n')]
+-
+-      # Check for opt-in analytics.
+-      if system_util.IsRunningInteractively() and collect_analytics:
+-        metrics.CheckAndMaybePromptForAnalyticsEnabling()
+-
+     if not args:
+       args = []
+ 
+@@ -414,117 +402,4 @@
+       ShutDownGsutil()
+     if GetFailureCount() > 0:
+       return_code = 1
+-    if command_changed_to_update:
+-      # If the command changed to update, the user's original command was
+-      # not executed.
+-      return_code = 1
+-      print('\n'.join(
+-          textwrap.wrap(
+-              'Update was successful. Exiting with code 1 as the original command '
+-              'issued prior to the update was not executed and should be re-run.'
+-          )))
+     return return_code
+-
+-  def MaybeCheckForAndOfferSoftwareUpdate(self, command_name, debug):
```

Comment:
Can we just patch in an early `return false` here?

I assume this would cause way less friction on future package bumps.